### PR TITLE
Jenkins CI

### PR DESCRIPTION
**Pull request by jkassemi**
Runs CI within docker container on Jenkins.

Once this is in place we'll continue to use Travis while we verify functionality and performance. Afterward we'll disable Travis and enable the deployment routine from the Jenkinsfile.

### CODE REVIEW

**Comment by robbiethegeek** [comment]
@aub @crwallace and @alexose seems like you folks should keep this in your purview

File: src/js/edu-benefits/1990/utils/validations.js [hunk 1]
```diff
@@ -274,6 +274,5 @@ export {
   isValidTourOfDuty,
   isValidEmploymentPeriod,
   isValidRotcScholarshipAmount,
-  isValidEducationPeriod,
-  isValidAddressField
```
Comment (crwallace) [inline src/js/edu-benefits/1990/utils/validations.js]: Should this be removed? This seems random.

**Review by crwallace — APPROVED** [review]
LGTM with one question

**Comment by aub** [comment]
This seems fine to me conceptually, though I'm with @crwallace on the random change to the validations. Is the idea that it'll be faster under Docker?

File: src/js/edu-benefits/1990/utils/validations.js (hunk 1 above)
Comment (jkassemi) [inline src/js/edu-benefits/1990/utils/validations.js]: @crwallace - it's a duplicate member. npm lint from the Jenkins docker-based tests caught the issue, but I haven't compared versions or config to determine why the version on Travis didn't. 

Removed from this and will submit in a new PR with better research.

**Pull request merged**